build.gradle.kts

```diff
@@ -20,7 +20,7 @@ val jaxwsVersion = "2.3.1"
 val jaxwsToolsVersion = "2.3.3"
 val kafkaClient = "7.0.1-ce"
 val confluentVersion = "7.0.1"
-val brukernotifikasjonSchemasVersion = "1.2021.01.18-11.12-b9c8c40b98d1"
+val brukernotifikasjonSchemasVersion = "v2.5.0"
```

Comment:
Vi kan gå for [v2.5.1](https://github.com/navikt/brukernotifikasjon-schemas/tree/v2.5.1) her